online-shop/src/app/auth/services/auth.service.ts

```diff
@@ -0,0 +1,43 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { httpOptions } from 'src/app/shared/constants';
+import { LoginCredentials } from 'src/app/shared/models/LoginCredentials.model';
+import { User } from '../models/User.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  private loginUrl = "http://localhost:3000/login"
```

Comment:
'http://localhost:3000' it's the base URL for the application
Base URL should be stored in the environments folder (environment.ts file). Base URL can be different for different environments (testing/production...)